.travis.yml

```diff
@@ -0,0 +1,10 @@
+install: true
+language: node_js
+node_js:
+  - "iojs"
+  - "7"
+# safelist
+branches:
+  only:
+  - master
+  - dev
```

Comment:
Don't we want to run tests on all branches except for gh-pages? Or perhaps this does something different.